lib/Swagger/src/Discussion/Models/SiteHalResponse.php

```diff
@@ -63,9 +63,9 @@ class SiteHalResponse implements ArrayAccess
       * @var string[]
       */
     protected static $swaggerTypes = array(
-        'shard_id' => 'int',
         'name' => 'string',
         'id' => 'int',
+        'shard_id' => 'int',
         '_links' => '\Swagger\Client\Discussion\Models\HalLinks',
         'requester_id' => 'string',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "requester_id" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)